src/Terminal.jl

```diff
@@ -53,16 +53,18 @@ This allows you to to see the messages from `println`, `dump`, `Pkg.status`, etc
 
 Example:
 
-```julia
-with_terminal() do
-    x=1+1
-    println(x)
-    @warn "Oopsie!"
-end 
+```jldoctest
+julia> with_terminal() do
+           x=1+1
+           println(x)
+           @warn "Oopsie!"
+       end
+PlutoUI.WithTerminalOutput("2\\n┌ Warning: Oopsie!\\n└ @ Main none:4\\n", "", nothing)
 ```
```

Comment:
I'm happy that you took the initiative to write tests, but these `jldoctest` tests are actually unhelpful, because:
1. We don't want to test the `text/plain` representation of PlutoUI results
2. Having these `text/plain` outputs in the docs is confusing for a number of reasons, the main one that this is not what you get when you run it in Pluto.

Reply:
jldoctest are helpful in ensure the output correct. But agree text/plain does not help. I just haven't found a better solution.